docs/querying/sql-functions.md

```diff
@@ -1227,35 +1269,146 @@ Returns a number for each output row of a groupBy query, indicating whether the
 
-**Function type:** [Scalar, sketch](sql-scalar.md#sketch-functions)
+* **Syntax:** `HLL_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS(expr, [numStdDev])`
+* **Function type:** Scalar, sketch
 
-Returns the distinct count estimate and error bounds from an HLL sketch.
+<details><summary>Example</summary>
+
+The following example estimates the number of unique tail numbers in the `flight-carriers` datasource with error bounds at plus or minus one standard deviation.
+
+```sql
+SELECT
+  HLL_SKETCH_ESTIMATE_WITH_ERROR_BOUNDS(DS_HLL("Tail_Number"), 1) AS "estimate_with_errors"
+FROM "flight-carriers"
+```
+
+Returns the following:
+
+| `estimate_with_errors` |
+| -- |
+| `[4685.8815405960595, 4536.952802775876, 4839.925808688653]` |
+
+
+</details>
+
+[Learn more](sql-scalar.md#sketch-functions)
 
 ## HLL_SKETCH_TO_STRING
 
-`HLL_SKETCH_TO_STRING(expr)`
+Returns a human-readable string representation of an HLL sketch for debugging. The `expr` argument must return an HLL sketch.
 
-**Function type:** [Scalar, sketch](sql-scalar.md#sketch-functions)
+* **Syntax:** `HLL_SKETCH_TO_STRING(expr)`
+* **Function type:** Scalar, sketch
+
+<details><summary>Example</summary>
+
+The following example returns the HLL sketch on column `Tail_Number` from the `flight-carriers` as a human-readable string.
+
+```sql
+SELECT
+  HLL_SKETCH_TO_STRING( DS_HLL("Tail_Number") ) AS "summary"
+FROM "flight-carriers"
+```
+
+Returns the following:
+
+<table>
+<tr>
+<td><code>summary</code></td>
+</tr>
+<tr>
+<td>
+
+```
+### HLL SKETCH SUMMARY: 
+  Log Config K   : 12
+  Hll Target     : HLL_4
+  Current Mode   : HLL
+  Memory         : false
+  LB             : 4611.381540678335
+  Estimate       : 4685.8815405960595
+  UB             : 4762.978259800803
+  OutOfOrder Flag: true
+  CurMin         : 0
+  NumAtCurMin    : 1316
+  HipAccum       : 0.0
+  KxQ0           : 2080.7755126953125
+  KxQ1           : 0.0
+  Rebuild KxQ Flg: false
+```
+
+</td>
+</tr>
+</table>
+
+
+</details>
+
+[Learn more](sql-scalar.md#sketch-functions)
 
-Returns a human-readable string representation of an HLL sketch.
 
 ## HLL_SKETCH_UNION
 
-`HLL_SKETCH_UNION([lgK, tgtHllType], expr0, expr1, ...)`
+Returns a union of HLL sketches, where each input expression must return an HLL sketch. The [HLL sketch documentation](../development//extensions-core//datasketches-hll.md) describes the optional `lgK` and `tgtHllType` arguments.
 
-**Function type:** [Scalar, sketch](sql-scalar.md#sketch-functions)
+* **Syntax:** `HLL_SKETCH_UNION([lgK, tgtHllType], expr0, expr1, ...)`
+* **Function type:** Scalar, sketch
+
+
+<details><summary>Example</summary>
 
-Returns a union of HLL sketches.
+The following example estimates the number of unique tail numbers that took off from California or Texas by estimating on the union of the HLL sketch of tail numbers that took off from `CA` and the HLL sketch of tail numbers that took off from `TX` from the `flight-carriers` datasource. 
```

Comment:
This sentence is too long. Consolidate the information or split the sentence into two to improve readability.